spago.dhall

```diff
@@ -2,29 +2,30 @@
 , license = "Apache-2.0"
 , repository = "https://github.com/purescript-halogen/purescript-halogen"
 , dependencies =
-    [ "aff"
-    , "avar"
-    , "console"
-    , "const"
-    , "dom-indexed"
-    , "event"
-    , "effect"
-    , "foreign"
-    , "fork"
-    , "free"
-    , "freeap"
-    , "halogen-vdom"
-    , "media-types"
-    , "nullable"
-    , "ordered-collections"
-    , "parallel"
-    , "profunctor"
-    , "transformers"
-    , "unsafe-coerce"
-    , "unsafe-reference"
-    , "web-file"
-    , "web-uievents"
-    ]
+  [ "aff"
+  , "avar"
+  , "console"
+  , "const"
+  , "debug"
+  , "dom-indexed"
+  , "effect"
+  , "event"
+  , "foreign"
+  , "fork"
+  , "free"
+  , "freeap"
+  , "halogen-vdom"
+  , "media-types"
+  , "nullable"
+  , "ordered-collections"
+  , "parallel"
+  , "profunctor"
+  , "transformers"
+  , "unsafe-coerce"
+  , "unsafe-reference"
+  , "web-file"
+  , "web-uievents"
+  ]
```

Comment:
This just adds the `debug` package (this doesn't change what users downstream receive when installing Halogen, as that's controlled by the package sets, so this is purely an internal change).